stable/database/README.md

```diff
@@ -238,6 +238,7 @@ The following tables list the configurable parameters of the `database` chart an
 | `sm.hotCopy.backupGroupPrefix` | Prefix for the automatically generated backup groups | `{{ .Values.cloud.cluster.name }}` |
 | `sm.hotCopy.backupGroups` | Backup groups configuration. By default a backup group per HCSM is created automatically | `{}` |
 | `sm.hotCopy.backupGroups.<name>.labels` | Space separated process labels used to select the Storage Managers which are part of this backup group. _Any_ label key and value should match for the SM to be selected | `nil` |
+| `sm.hotCopy.backupGroups.<name>.processFilter` | LBQuery expression used to select the Storage Managers which are part of this backup group. For more information on LBQuery expression syntax, see <https://doc.nuodb.com/nuodb/latest/client-development/load-balancer-policies/#lbquery-expression-syntax>. | `nil` |
```

Comment:
Can you use a link with text so that it does not fill up the field? e.g.
```
For more information, see [LBQuery expression syntax](https://doc.nuodb.com/nuodb/latest/client-development/load-balancer-policies/#lbquery-expression-syntax).
```

Reply:
Good point!